crates/sui-core/src/authority.rs

```diff
@@ -354,6 +355,12 @@ const LOW_LATENCY_SEC_BUCKETS: &[f64] = &[
     0.2, 0.5, 1., 2., 5., 10., 20., 50., 100.,
 ];
 
+// Buckets for consensus timestamp bias in seconds.
+const TIMESTAMP_BIAS_SEC_BUCKETS: &[f64] = &[
```

Comment:
I would expect most of the "time" bias would be 3-4 consensus rounds, or 200-300ms. When system time is less advanced, it should be in the order of hundreds of ms too. Do we want to put finer grained buckets around that?